concepts/standard-liquidity-providers/README.md

```diff
@@ -15,30 +15,32 @@ description: Insuring the insurance (Hedging Agents)
 
 Standard Liquidity Providers are the buffer for the moments when hedging agents do not fully cover the collateral that was brought by users.
 
-They entrust Angle with their liquidity and like liquidity providers in other protocols (Compound, Uniswap, Aave), they automatically accrue interests on the assets they brought. The risk for them is to incur a slippage when the protocol is not enough collateralized and they want to cash out.
+They entrust Angle with their liquidity and like liquidity providers in other protocols (Compound, Uniswap, Aave), they automatically accrue interest on the assets they brought. The risk for them is to incur a slippage when the protocol is not enough collateralized and they want to cash out.
 
 ## 🎁 Rewards
 
 There are different incentives for SLPs to come to Angle:
 
 - A fraction of the transaction fees induced by users minting and burning stable assets are redistributed to SLPs in proportion to how they contribute to the protocol.
-- At each point in time, the protocol owns reserves which are only useful when redeemed by a user, a SLP or an HA. Part of the reserves is automatically transferred to strategies (like Yearn strategies) responsible for getting yield on it by, for instance, lending to protocols like Compound or Aave. SLPs gets a fraction of the returns made by these strategies.
+- At each point in time, the protocol owns reserves which are only useful when redeemed by a user, a SLP or an HA. Part of the reserves is automatically transferred to strategies (like Yearn strategies) responsible for getting yield on it by, for instance, lending to protocols like Compound or Aave. SLPs gets a fraction of the interest made by these strategies.
 
-The fraction of transaction fees and the fraction of lending returns going to SLPs correspond to two different parameters controlled by governance.
+Note that SLPs are also able to stake their positions (in the form of sanTokens) to receive ANGLE rewards.
 
-Note that SLPs are also able to stake their positions (in the form of sanTokens) to receive governance tokens.
+{% hint style="info" %}
+The fraction of transaction fees and interest going to SLPs correspond to two different parameters controlled by governance: feesForSLPs, and interestsForSLPs. InterestsForSLPs is what they get **after** taking into account the share going to veANGLE holders (interestsForSurplus). The value of these parameters can be seen in [Angle's SDK Parameters file](https://github.com/AngleProtocol/angle-sdk/blob/main/src/constants/parameters/mainnet.ts#L66). Note that they can be specific for each collateral.
```

Comment:
Should point to analytics in this user friendly docs, as such no need to name the exact parameters